configuration/configuring-root-certificates/en.md

```diff
@@ -0,0 +1,119 @@
++++
+title = "Configuring Root Certificates"
+lastmod = "2022-06-10T14:22:00-06:00"
++++
+# Configuring Root Certificates
+
+Mange local SSL/TLS Certificate Authority (CA) certificates with any of the desktop environments supported by Solus.
+
+## Public vs. Local Certificates
+
+Public SSL certiicates are included in Solus system packages and are regularly updated with normal package updates.
+
+There are many reasons you may want to install a custom, self-signed, or certificate provided by your local systems administrator.
```

Comment:
Grammatical note:
"install a custom, self-signed, or certificate" 
Expanded this translates to a custom certificate, a self-signed certificate or a certificate certificate. Better wording might be

"There are many reasons you may want to install a certificate that is custom, self-signed, or provided by your local systems administrator."

Reply:
Agreed and updated!